packages/checkout/widgets-lib/src/lib/constants.ts

```diff
@@ -105,3 +105,5 @@ export const PASSPORT_URL = {
   [Environment.SANDBOX]: 'https://passport.sandbox.immutable.com/',
   [Environment.PRODUCTION]: 'https://passport.immutable.com/',
 };
+
+export const WITHDRAWAL_CLAIM_GAS_LIMIT = 91000;
```

Comment:
Why is this needed? Or, shouldn't this be part of the Bridge SDKs?

Reply:
I've added it in case our estmateGas call fails.

Reply:
is that limit the same for Testnet and Mainnet?